source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container-dark.css

```diff
@@ -20,36 +20,36 @@
 }
 
 .berlin-large-container {
-    height: 462px;
+    height: 464px;
 }
 
 .berlin-medium-container {
-    height: 304px;
+    height: 306px;
 }
 
 .berlin-small-container {
-    height: 146px;
+    height: 148px;
 }
 
 
 /* berlin-large/medium/small-container  4px(top+bottom padding+margin) berlin-outer-container border*/
 
 .berlin-large-card {
-    height: 462px;
+    height: 398px;
 }
 
 .berlin-medium-card {
-    height: 304px;
+    height: 240px;
 }
 
 .berlin-small-card {
-    height: 146px;
+    height: 82px;
 }
 
 .ac-adaptiveCard {
     border: 0px;
-    padding: 48px 16px 16px 16px !important;
-    border-radius: 8px !important;
+    margin: 48px 16px 16px 16px !important;
+    padding: 0px !important;
```

Comment:
are these changes up-to-date with the latest design?

Reply:
@paulcam206 

Yes. I switched to margins because overflow still allows content within padding, so we would allow something like this:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/98650930/190827158-ff10a041-2162-4f18-b327-4e32215f3737.png">
where the highlighted text is within the margin and it should not be displayed in preview mode.

Now, we cut off the content for the margin:
<img width="203" alt="image" src="https://user-images.githubusercontent.com/98650930/190827525-1b00ce9d-00c3-4cd8-8624-c90c60c7d41c.png">


As for the sizing changes, I confirmed with Amanda that the provided size does **not** include the border (1px), so we had to increase all of our sizes by 2px.

Before:
<img width="118" alt="image" src="https://user-images.githubusercontent.com/98650930/190827347-2891ba65-e5e6-42cb-ae29-6bf42bd1bc3c.png">

After: 
<img width="119" alt="image" src="https://user-images.githubusercontent.com/98650930/190827409-961925a8-674c-4181-b439-073e069d7a5e.png">